### PR TITLE
MB 4436 Standardize Heading type/size across all move setup screens

### PR DESCRIPTION
**Pull request by Shkeating**
## Description

This branch makes it so all top-level headers in the customer app are h1s, with the same text size, weight, and type scaling across devices. This creates a more consistent page layout across the customer app experience.

The pages affected by this PR include:
   - ppmCloseout - Customer Agreement page
   - hhg - HHG details form, Review Your Details, “Now for the official part” pages
   - Homepage “Welcome to my.move.mil”
   - ppm homepage “New move”
   - ppm “How much do you think you’ll move” page

## Reviewer Notes

Is there any pages I missed? Let me know! I caught most of these by running through our e2e/integration tests, so if there are areas they don't cover I'll have to go back through and look.

## Setup

```sh
make db_dev_e2e_populate
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4436?atlOrigin=eyJpIjoiOTk4MjJhN2YyOWYzNGVhMzlkZDA0NTVmYTdmNmU5MzgiLCJwIjoiaiJ9) for this change

### CODE REVIEW

**Review by kimallen — COMMENTED** [review]
Looks like parts of the Payment request flow needs those changes too  `scenes/Moves/Ppm/PpmPaymentRequestIntro`,  `WeightTicketExamples.jsx`
That may not be all-inclusive.  I recommend checking in `scenes/MyMove/index.jsx` Which leads you to all of the pages that would have toplevel H1s.  Also look through `getWorkFlowRoutes.jsx` which also names pages.

File: src/components/Customer/EditShipment.jsx [hunk 1]
```diff
@@ -283,9 +283,9 @@ class EditShipment extends Component {
     return (
       <div className="grid-container">
         <div className={`margin-top-2 ${styles['hhg-label']}`}>{`HHG ${!!shipmentNumber && shipmentNumber}`}</div>
-        <h2 className="margin-top-1" style={{ fontSize: 28 }}>
+        <h1 className="margin-top-1" style={{ fontSize: 28 }}>
```
Comment (suzubara) [inline src/components/Customer/EditShipment.jsx:286]: @Shkeating Should this still have a custom font size applied with the style attr?

Comment (Shkeating) [reply src/components/Customer/EditShipment.jsx:286]: nope!! thanks for pointing that out. I am seeing several of these have font sizes getting applied to them that I need to adjust. I didn't realize that making a draft PR would still notify everyone 😅

**Comment by Shkeating** [comment]
@amanda5iu some images:

before on the left, after on the right:

**HHG Details, mobile**
image


**HHG Details, desktop**
image

**Review by suzubara — CHANGES_REQUESTED** [review]
could you remove the `.loki` directory from your branch? Updating from master might also resolve it. otherwise the changes look good!

**Review by amanda5iu — APPROVED** [review]
So satisfying to see this. LGTM!

**Review by suzubara — APPROVED** [review]

**Pull request merged**